### PR TITLE
What_are_Classes_&_Object_Programming(A).md

### LINKED ISSUE #4025: What_are_Classes_&_Object_Programming(A).md
### Description

Classes and Object Programming 
(intro to OOPS)
Complete Beginner to Advance Audio Series.

### Domain

C/CPP

### Type of Contribution

Audio

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by miraj0507**
<hr>

## Description 📜
INTRODUCTION TO OOPS_WHAT IS THE NEED OF OOPS_EXPLAINED WHAT ARE CLASSES AND OBJECTS
Fixes #4025

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [x] C/#CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# WHAT ARE CLASSES AND OBJECT PROGRAMMING? WHY THERE WAS A NEED FOR OOPS
```
Comment (kashishahuja2002) [inline C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md:1]: "WHY THERE WAS A NEED FOR OOPS?" This should be in next line.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add a little content before the audio file.
Don't write everything in uppercase.

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md (hunk 1 above)
Comment (miraj0507) [reply C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md]: Did resolve the issue, 
Thank you,
Do review

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# WHAT ARE CLASSES AND OBJECT PROGRAMMING? 
+WHY THERE WAS A NEED FOR OOPS,
```
Comment (kashishahuja2002) [inline C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md:2]: Put ? at end.

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+The Idea of Classes and Objects are described and the audio lays a firm foundation about the need of classes,
+A real life senario is discussed along with the properties.  
+[GETTING STARTED OOPS,UNDERSTAND THE NEED OF OOPS IN C++](https://drive.google.com/file/d/1QXs1aGzYMKYtwntDsD2gFsNM7m2J2SXr/view?usp=sharing)
```
Comment (kashishahuja2002) [inline C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md:5]: Change it to lower case.

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+
+
+
+
+
```
Comment (kashishahuja2002) [inline C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md:11]: Remove these lines.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md (hunk 4 above)
Comment (miraj0507) [reply C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md]: Thank you can you please now check

File: C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# WHAT ARE CLASSES AND OBJECT PROGRAMMING? 
+WHY THERE WAS A NEED FOR OOPS?
+The Idea of Classes and Objects are described and the audio lays a firm foundation about the need of classes,
+A real life senario is discussed along with the properties.  
+[Getting started with oops? Understand the foundation.](https://drive.google.com/file/d/1QXs1aGzYMKYtwntDsD2gFsNM7m2J2SXr/view?usp=sharing)
```
Comment (kashishahuja2002) [inline C_CPP/OOPs in C++/Classes & Objects Programming/What_are_Classes_&_Object_Programming(A).md:5]: Add an empty line before this and write it as "Audio Link: Getting star......."

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Review by kashishahuja2002 — APPROVED** [review]
LGTM @Saviour1001

**Pull request merged**